src/components/RecipeList.tsx

```diff
@@ -91,42 +94,63 @@ const RecipeList: React.FC<RecipeListScreenProps> = ({ navigation }) => {
       <View style={styles.listItem}>
         <Image style={styles.image} source={kalaseljankaImg} />
         <View style={styles.listItemContentArea}>
-          <Text style={styles.listItemHeader}>{item.name}</Text>
-          <Text>Rating: {item.rating}</Text>
+          <Text style={[textStyles.headingSmall, { marginBottom: 4 }]}>
+            {item.name}
+          </Text>
+          {/* <View style={{height: 4}}></View> */}
+          <Text style={[textStyles.bodyMedium, colorStyles.grey60]}>
+            Rating: {item.rating}
+          </Text>
         </View>
       </View>
     </Pressable>
   );
   return (
-    <View style={styles.recipeList}>
-      <Text style={styles.heading}>My recipes</Text>
-      <FlatList data={recipes} renderItem={renderRecipeListItem} />
+    <View>
+      <View style={styles.header}>
+        <Text style={textStyles.headingLarge}>My recipes</Text>
+        <Pressable>
+          <AntDesign name="user" size={32} color="black" />
+        </Pressable>
+      </View>
+
+      <View style={styles.recipeList}>
+        <FlatList data={recipes} renderItem={renderRecipeListItem} />
+      </View>
     </View>
   );
 };
 
 const styles = StyleSheet.create({
   recipeList: {
-    margin: 24,
+    marginTop: 8,
+    marginLeft: 24,
+    marginRight: 24,
   },
-  heading: {
-    fontSize: 18,
-    fontWeight: 'bold',
-    marginBottom: 24,
+  header: {
+    margin: 24,
+    marginTop: 72,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
   },
   listItem: {
+    borderColor: '#666666',
+    borderWidth: 1,
     flexDirection: 'row',
-    marginBottom: 8,
+    marginBottom: 16,
+    // minHeight: 80,
```

Comment:
Remove commented-out lines, if there's no clear reason for them.

Reply:
✅